src/model/NPMResolver.ts

```diff
@@ -30,7 +30,7 @@ export class NPMResolver {
     }
 
     public async resolve(): Promise<void> {
-        if (!PackageVersion.getOrNull()) {
+        if (!PackageVersion.get()) {
```

Comment:
Wait ... this case cannot happen, right?
`get()` will throw an error or return a valid version.

Reply:
Thank you - removed the `if` 🙈